exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -395,6 +406,22 @@ public Viewable getProfile(@PathParam("queryid") String queryId){
     }
   }
 
+  @POST
+  @Path("/profiles/view")
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  @Produces(MediaType.TEXT_HTML)
+  public Viewable viewProfile(@FormDataParam("profileFile") String content){
```

Comment:
It is possible to rename the "profileFile" to "profile_object" or "profile_data" ... ?